friends/src/App.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import { Route, Link } from 'react-router-dom';
+
+import Friends from './components/friends';
+import AddFriend from './components/addFriend';
+
+// server url
+let url = 'http://localhost:5000/friends';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state= {
+      data: [],
+      inputName: '',
+      inputAge: '',
+      inputEmail: '',
+    }
+  }
+
+  // request data from server and setState with response
+  componentDidMount() {
+    axios
+      .get(url)
+      .then(res => {
+        this.setState({
+          data: res.data,
+        });
+      })
+      .catch(err => {
+        console.log(err);
+      })
```

Comment:
This callback is just taking your error and calling console.log with it. Why not just use console.log as your callback?
```javascript
// ...
.catch(console.log);
```